components/installation.tsx

```diff
@@ -0,0 +1,213 @@
+"use client";
+import React, { useState } from "react";
+import Image from "next/image";
+import Snackbar from "@mui/material/Snackbar";
+import CloseRoundedIcon from "@mui/icons-material/CloseRounded";
+import IconButton from "@mui/material/IconButton";
+import CopyButton from "@/public/images/icons8-copy-96.png";
+import SvgIcon from "@mui/material/SvgIcon";
+import { IconButton as MuiButton } from "@mui/material";
+import { red } from "@mui/material/colors";
+
+const Installation = () => {
+  const [checkedIndex, setCheckedIndex] = useState<number | null>(null);
+  const [open, setOpen] = React.useState(false);
+
+  const handleClose = (
+    event: React.SyntheticEvent | Event,
+    reason?: string
+  ) => {
+    if (reason === "clickaway") {
+      return;
+    }
+
+    setOpen(false);
+  };
+  const action = (
+    <React.Fragment>
+      <MuiButton color={"success"} onClick={handleClose}>
+        <SvgIcon component={CloseRoundedIcon} inheritViewBox />
+      </MuiButton>
+    </React.Fragment>
+  );
+
+  const handleclick = (index: number) => {
+    let codeToCopy = "";
+    switch (index) {
+      case 0:
+        codeToCopy =
+          "curl -O https://raw.githubusercontent.com/keploy/keploy/main/keploy.sh && source";
+        break;
+      case 1:
+        codeToCopy = "keploy record -c CMD_TO_RUN_APP";
+        break;
+      case 2:
+        codeToCopy = "keploy test -c CMD_TO_RUN_APP --delay 10";
+        break;
+      default:
+        break;
+    }
+
+    if (codeToCopy !== "") {
+      navigator.clipboard.writeText(codeToCopy).catch((error) => {
+        console.error("Error copying code to clipboard: ", error);
+      });
+    }
+
+    setOpen(true);
+
+    setCheckedIndex(index);
+    setTimeout(() => {
+      setCheckedIndex(null);
+    }, 2000); // 2000 milliseconds = 2 seconds
+  };
+
+  return (
+    <div className="max-w-6xl mx-auto px-4 sm:px-6 mb-10 ">
+      {/* Hero content */}
+      <div className="pt-30 pb-12 md:pt-30 md:pb-20 rounded-lg p-10 gap-16">
+        {/* Section header */}
+        <div className="text-center pb-10 md:pb-10">
+          <h1 className="h2 mb-4 text-secondary-300" data-aos="zoom-y-out">
+            Getting Started
+            <br />
+          </h1>
+          <h1 className="text-xl text-gray-600" data-aos="zoom-y-out">
+            Start Recording Test cases with code-less integration
+            <br />
+          </h1>
+        </div>
+
+        <div
+          className="max-w-4xl mx-auto pb-12 md:pb-16 "
+          data-aos="zoom-y-out"
+        >
+          <h1 className="text-2xl text-secondary-300 font-semibold leading-tighter tracking-tighter mb-2 ">
```

Comment:
the margin is inconsistent please change it to mb-4

Reply:
hey resolved it!.